matter_server/server/server.py

```diff
@@ -134,7 +146,7 @@ async def start(self) -> None:
         self.loop = asyncio.get_running_loop()
         self.loop.set_exception_handler(_global_loop_exception_handler)
         self.loop.set_debug(os.environ.get("PYTHONDEBUG", "") != "")
-        await self.device_controller.initialize()
+        await self.device_controller.initialize(self.paa_root_cert_dir)
```

Comment:
Did you check if this actually works now ? In the past this did not work as it was hardcoded somewhere. Did you commission a new device ?

Reply:
Yes, tested by commissioning a new device (WiZ), worked with an alternative path :+1: 

```
...
2024-03-07 20:55:14 (Dummy-2) CHIP_PROGRESS [chip.native.CTL] Successfully finished commissioning step 'AttestationVerification'
...
```